packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -513,42 +517,48 @@ class ServiceExtensionManager {
     }
     final Isolate isolate = await _service.getIsolate(isolateRef.id);
     if (isolate.extensionRPCs != null) {
-      for (String extension in isolate.extensionRPCs) {
-        await _maybeAddServiceExtension(extension);
-      }
+      if (await connectedApp.isAnyFlutterApp) {
+        for (String extension in isolate.extensionRPCs) {
+          await _maybeAddServiceExtension(extension);
+        }
 
-      if (_pendingServiceExtensions.isEmpty) {
-        extensionStatesUpdated.complete();
-      }
+        if (_pendingServiceExtensions.isEmpty) {
+          extensionStatesUpdated.complete();
+        }
 
-      if (!_firstFrameEventReceived) {
-        bool didSendFirstFrameEvent = false;
-        if (isServiceExtensionAvailable(extensions.didSendFirstFrameEvent)) {
-          final value = await _service.callServiceExtension(
-            extensions.didSendFirstFrameEvent,
-            isolateId: _isolateManager.selectedIsolate.id,
-          );
-          didSendFirstFrameEvent =
-              value != null && value.json['enabled'] == 'true';
-        } else {
-          final EvalOnDartLibrary flutterLibrary = EvalOnDartLibrary(
-            [
-              'package:flutter/src/widgets/binding.dart',
-              'package:flutter_web/src/widgets/binding.dart',
-            ],
-            _service,
-          );
-          final InstanceRef value = await flutterLibrary.eval(
-            'WidgetsBinding.instance.debugDidSendFirstFrameEvent',
-            isAlive: null,
-          );
+        if (!_firstFrameEventReceived) {
+          bool didSendFirstFrameEvent = false;
+          if (isServiceExtensionAvailable(extensions.didSendFirstFrameEvent)) {
+            final value = await _service.callServiceExtension(
+              extensions.didSendFirstFrameEvent,
+              isolateId: _isolateManager.selectedIsolate.id,
+            );
+            didSendFirstFrameEvent =
+                value != null && value.json['enabled'] == 'true';
+          } else {
+            final EvalOnDartLibrary flutterLibrary = EvalOnDartLibrary(
+              [
+                'package:flutter/src/widgets/binding.dart',
+                'package:flutter_web/src/widgets/binding.dart',
```

Comment:
Is this still necessary? We've deprecated package:flutter_web now that it's been merged into the main framework.

Reply:
removed